__tests__/app.test.js

```diff
@@ -134,6 +146,44 @@ describe('GET /api/articles?order', () => {
   });
 });
 
+describe('GET /api/articles?topic', () => {
+  test('200: Responds with an array of articles matching topic query', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(4);
+        articles.forEach((article) => {
+          expect(article.topic).toBe('mitch');
+        });
+      });
+  });
+  test('200: Responds with an empty array if topic exists, but no articles match topic query', () => {
+    return request(app)
+      .get('/api/articles?topic=paper')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(0);
+      });
```

Comment:
Really good test, if the topic doesn't exist a valid response is 200 with `[]` as that ___is___ everything that currently matches the topic 🥇